lms/djangoapps/commerce/views.py

```diff
@@ -10,6 +10,7 @@
 from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification
 from shoppingcart.processors.CyberSource2 import is_user_payment_error
 from django.utils.translation import ugettext as _
+from openedx.core.djangoapps.theming import helpers
```

Comment:
@douglashall @mattdrayer I have incorporated it in PR https://github.com/edx/edx-platform/pull/12416